stacker/providers/aws.py

```diff
@@ -183,3 +183,30 @@ def get_outputs(self, stack_name, *args, **kwargs):
             stack = self.get_stack(stack_name)
             self._outputs[stack_name] = get_output_dict(stack)
         return self._outputs[stack_name]
+
+    def get_stack_info(self, stack_name):
+        """ Get the template and parameters of the stack currently in AWS
+
+        Returns [ template, parameters ]
+        """
+        try:
+            stacks = retry_on_throttling(
+                self.cloudformation.describe_stacks,
+                kwargs=dict(stack_name_or_id=stack_name))
+        except boto.exception.BotoServerError as e:
+            if 'does not exist' not in e.message:
+                raise
+            raise exceptions.StackDoesNotExist(stack_name)
+
+        stack = stacks[0]
+        parameters = dict()
+        for p in stack.parameters:
+            parameters[p.key] = p.value
+        ret = stack.get_template()
+        try:
```

Comment:
Is this actually valid?  The stack already exists - otherwise it would have failed much higher than this, right?